examples/demo-stf/src/runtime.rs

```diff
@@ -124,27 +125,28 @@ impl<C: Context, Cond: ValidityCondition> SlotHooks<Cond> for Runtime<C> {
     }
 }
 
-impl<C, Cond, B> sov_modules_stf_template::Runtime<C, Cond, B> for Runtime<C>
+impl<C, Da> sov_modules_stf_template::Runtime<C, Da> for Runtime<C>
 where
     C: Context,
-    Cond: ValidityCondition,
-    B: BlobReaderTrait,
+    Da: DaSpec,
 {
 }
 
-impl<C: Context> BlobSelector for Runtime<C> {
+impl<C: Context, Da: DaSpec> BlobSelector<Da> for Runtime<C> {
```

Comment:
Ouch, I only now noticed, that I introduced similar, but different change: https://github.com/Sovereign-Labs/sovereign-sdk/pull/759 . It looks like having `Da` as generic is a little better, since anything else from this trait can be also accessed